dissect/target/loaders/log.py

```diff
@@ -1,13 +1,21 @@
+from __future__ import annotations
+
 import urllib
 from pathlib import Path
 from typing import Union
 
 from dissect.target import Target
-from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.filesystem import RootFilesystem, VirtualFilesystem
 from dissect.target.loader import Loader
+from dissect.target.plugin import arg
+
+logloader_option_hint = "hint"
 
 
+@arg("--log-hint", dest=logloader_option_hint, help="Hint for file type")
 class LogLoader(Loader):
+    """Load separate log files without a target. Usage: target-query /evtx/* -L log -f evtx (-L log -h for options)"""
```

Comment:
```suggestion
    """Load separate log files without a target. Usage: target-query /evtx/* -L log -f evtx (-L log -h for options)."""
```
Having this long line with the usage in it looks a bit weird, but I guess you do that for the `-l` output?